### PR TITLE
Netvox

### PR DESCRIPTION
**Pull request by J2thatsme**
Added netvox button,r711, door & water leakage

### CODE REVIEW

File: types/netvox/water_leakage/uplink.spec.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+        if (value.topic === "lifecycle") {
+          assert.equal(value.data.voltage, 2.8);
+
+          validate(value.data, lifecycleSchema, { throwError: true });
+        }
+      });
+
+      consume(data);
+      done();
```
Comment (mariusgiger) [inline types/netvox/water_leakage/uplink.spec.js:71]: done can be removed

Comment (J2thatsme) [reply types/netvox/water_leakage/uplink.spec.js:71]: I get a timeout if i do that

Comment (mariusgiger) [reply types/netvox/water_leakage/uplink.spec.js:71]: you also have to remove the method parameter (done)

Comment (J2thatsme) [reply types/netvox/water_leakage/uplink.spec.js:71]: i see thx

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**